caesar_cipher.rb

```diff
@@ -0,0 +1,97 @@
+# Program that will cipher a String that the user will pass to it.
+
+class Caesar
+    ALPHABET_UPPERCASE = Array ('A'..'Z')
+    ALPHABET_LOWERCASE = Array ('a'..'z')
+
+    def initialize(text, num)
+        @text = text
+        @num = num
+    end
+
+    # Method that ciphers the text that we give it
+    def cipher
+        # It will check if there is text to cipher and if there is a valid number to change the letters
+        return puts "The text given is null and/or the number given is 0." if text == "" || num.zero?
```

Comment:
What will the program return if the user passes an blank string ``' '``? 
IMO, you should check the case where the string is blank. 
```suggestion
        return puts "The text given is null and/or the number given is 0." if text.strip.empty? || num.zero?
```
Please check how it works in your ruby console(`irb`) 
DOCS for strip method: https://reintech.io/blog/exploring-the-string-strip-method-in-ruby

Reply:
+1